vJoySerialFeeder/MainForm.cs

```diff
@@ -57,7 +57,8 @@ public partial class MainForm : Form
 										typeof(SbusReader),
 										typeof(FportReader),
 										typeof(DsmReader),
-										typeof(DummyReader)
+										typeof(DummyReader),
+										typeof(CrsfReader)
```

Comment:
Can you, please, put your reader above `DummyReader`. I would like dummy to be the last in the list (the same change in `MainForm.Designer.cs` is needed).